src/api-reference/request/v4.endpoints.schemas.markdown

```diff
@@ -200,16 +199,16 @@ Name|Type|Format|Description
 
 Name|Type|Format|Description
 ---|---|---|---
-`city`|`string`|-|**Required if `city` is used** The city name of the location.
+`city`|`string`|-|The city name of the location. Always provide the `countryCode` value in addition to the `city`.
```

Comment:
Please revert this to its original state. This is the standard way we document this function across the entire site.